#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_watch.1.dpatch by  <csmall@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Cleanup of manual page
## DP: cumulative is a parameter not option #527193

@DPATCH@
diff -urNad procps-3.2.8~/watch.1 procps-3.2.8/watch.1
--- procps-3.2.8~/watch.1	2009-05-11 22:40:13.000000000 +1000
+++ procps-3.2.8/watch.1	2009-05-11 22:46:33.000000000 +1000
@@ -1,45 +1,54 @@
-.TH WATCH 1 "1999 Apr 3" " " "Linux User's Manual"
+.TH WATCH 1 "2009 May 11" " " "Linux User's Manual"
 .SH NAME
 watch \- execute a program periodically, showing output fullscreen
 .SH SYNOPSIS
+.na
 .B watch
-.I [\-dhvt] [\-n <seconds>] [\-\-differences[=cumulative]] [\-\-help] [\-\-interval=<seconds>] [\-\-no\-title] [\-\-version] <command>
+.RB [ \-dhvt ]
+.RB [ \-n
+.IR seconds ]
+.RB [ \-\-differences[=\fIcumulative\fP]]
+.RB [ \-\-help ]
+.RB [ \-\-interval=\fIseconds\fP]
+.RB [ \-\-no\-title ]
+.RB [ \-\-version ]
+.I command
 .SH DESCRIPTION
-.BR watch
+.B watch
 runs
 .I command
 repeatedly, displaying its output (the first screenfull).  This allows you to
 watch the program output change over time.  By default, the program is run
 every 2 seconds; use 
-.I -n
+.B \-n
 or
-.I --interval
+.B \-\-interval
 to specify a different interval.
 .PP
 The
-.I -d
+.B \-d
 or
-.I --differences
-flag will highlight the differences between successive updates.  The 
-.I --cumulative
-option makes highlighting "sticky", presenting a running display of all
+.B \-\-differences
+flag will highlight the differences between successive updates.  Using
+.B \-\-differences=\fIcumulative\fP
+makes highlighting "sticky", presenting a running display of all
 positions that have ever changed.  The
-.I -t
+.B \-t
 or
-.I --no-title
+.B \-\-no\-title
 option turns off the header showing the interval, command, and current
 time at the top of the display, as well as the following blank line.
 .PP
-.BR watch
+.B watch
 will run until interrupted.
 .SH NOTE
 Note that
 .I command
-is given to "sh -c"
+is given to "sh \-c"
 which means that you may need to use extra quoting to get the desired effect.
 .PP
 Note that POSIX option processing is used (i.e., option processing stops at
-the first non-option argument).  This means that flags after
+the first non\-option argument).  This means that flags after
 .I command
 don't get interpreted by
 .BR watch
@@ -61,20 +70,20 @@
 To see the effects of quoting, try these out
 .IP
 watch echo $$
-.IP
+.br
 watch echo '$$'
-.IP
+.br
 watch echo "'"'$$'"'"
 .PP
 You can watch for your administrator to install the latest kernel with
 .IP
-watch uname -r
+watch uname \-r
 .PP
 (Just kidding.)
 .SH BUGS
 Upon terminal resize, the screen will not be correctly repainted until the
 next scheduled update.  All
-.I --differences
+.B \-\-differences
 highlighting is lost on that update as well.
 .PP
 Non-printing characters are stripped from program output.  Use "cat -v" as
